src/components/PageSection.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
```

Comment:
I think the latest version can do without this line

Reply:
fixed in commit d169cbc